api/src/job.js

```diff
@@ -466,9 +442,13 @@ class Job {
         this.logger.info(`Cleaning up job`);
 
         this.exit_cleanup(); // Run process janitor, just incase there are any residual processes somehow
+        this.close_cleanup();
         await this.cleanup_filesystem();
 
         remaining_job_spaces++;
+        if (job_queue.length > 0) {
+            job_queue.shift()();
```

Comment:
double brackets on this shift?

Reply:
Yeah we are calling the shifted `resolve` function. It was like this previously :D